### PR TITLE
Added link back to login, fixed bug with loading calander from "All batches view" as a role 3 user

### PR DESCRIPTION
**Pull request by bmckalip**
1. added link to login page on registration page

2. fixed typo on edit batch page in the navbar

3. fixed calander not loading problem when viewing as QC

more info on 3:
problem was that it was trying to load a calendar by trainerBatch.id, but this doesn't exist. however, the currentBatch does, so I just changed it to use that instead. Not sure if this is a good way to fix this or not because of lack of knowledge of the program. 

tldr: if roletype == 2 || roleType == 3 then use currentBatch instead of trying to use currentBatch for role 2 and trainer batch for role 3

### CODE REVIEW

**Comment by tdnguyen0106** [comment]
Got rid of the merge conflict and an empty "else" statement that you had left behind.
